app/services/articles/feeds/basic.rb

```diff
@@ -1,7 +1,7 @@
 module Articles
   module Feeds
     class Basic
-      def initialize(user: nil, number_of_articles: 25, page: 1, tag: nil)
```

Comment:
This is a notable change for the `Articles::Feeds::Basic`, could someone provide context as to why I may not want this to be 50?  Perhaps @benhalpern or someone from SRE?